src/apps/material/material.test.ts

```diff
@@ -0,0 +1,78 @@
+describe("Material app e2e", () => {
```

Comment:
```suggestion
describe("Material", () => {
```

There is no reason to add app or e2e to the test name. This is not something we do for other tests as well.